pyproject.toml

```diff
@@ -1,17 +1,16 @@
 [build-system]
-requires = ["hatchling"]
-build-backend = "hatchling.build"
+requires = ["uv_build>=0.9.17,<0.10.0"]
+build-backend = "uv_build"
```

Comment:
I don't have any experience with UV build system, but it seems like to expect an `src` directory in the root:
https://docs.astral.sh/uv/concepts/build-backend/#modules

We might need to change that configuration as it describes with `module-root`

Reply:
```toml
[tool.uv.build-backend]
module-name = "sqladmin"
module-root = ""
```